### PR TITLE
Meeting minutes for 2021-08-31

### PR DESCRIPTION
**Pull request by ChristianKuehnel**

### CODE REVIEW

File: meeting_minutes/2021-08-31 [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  * Come to a decision within the community.
+  * Maybe run an RFC around code reiviews (with a structured timeline) 
+    and get community feedback. Make a decision after the review peroid.
+    * What would be a good time for that? Maybe wait until the 
+      bugzilla/issue migration is coming to an end?
+
+## Releasing compilers
+
+* Is anyone creating LLVM releases off the main branch? Please let @christianKuehnel know. 
+  We might want to join forces on building tools for that.
```
Comment (tstellar) [inline meeting_minutes/2021-08-31:55]: Sorry I had to drop early, but I am interested in this.

File: meeting_minutes/2021-08-31 [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  * What are users experience during the review process?
+* Open points: 
+  * How do we convince the community? Why do we thing this is the right tool?
+  * What is the biggest push-back from the community?
+    * Review interface is not as feature-rich as Phabricator.
+    * Reviewing stacked patches is more ugly.
+    * We can't influence GitHub's roadmap.
+    * The GitHub UI is fairly slow in China.
+  * Come to a decision within the community.
+  * Maybe run an RFC around code reiviews (with a structured timeline)
```
Comment (joker-eph) [inline meeting_minutes/2021-08-31:47]:
```suggestion
  * Maybe run an RFC around code reviews (with a structured timeline) 
```

File: meeting_minutes/2021-08-31 [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  * see also [our last meeting](https://github.com/llvm/llvm-iwg/blob/main/meeting_minutes/2021-08-17.md#update-on-bugzilla-migration)
+  * Timeline for bugzilla/issue migration: next few months
+* We need to differentiate two topics, to be discussed separately:
+  * What is actually technically needed for Pull Requests? (checking CLA, licences, issue IDs,...)
+  * What are users experience during the review process?
+* Open points: 
+  * How do we convince the community? Why do we thing this is the right tool?
+  * What is the biggest push-back from the community?
+    * Review interface is not as feature-rich as Phabricator.
+    * Reviewing stacked patches is more ugly.
```
Comment (joker-eph) [inline meeting_minutes/2021-08-31:43]: "more ugly" is an under-statement: you could say "not supported".

Something to add to the list here is: "lack of Herald rules". When we had a round-table on PullRequest at the dev mtg in 2019 this was considered a hard blocker by a few. After that I searched online but couldn't find any alternative readily usable for GitHub. This is something that might be possible to implement ourselves with bots polling GitHub and managing the subscription, but this needs "work".

**Review by lazyparser — APPROVED** [review]
LGTM. Thanks.

File: meeting_minutes/2021-08-31 (hunk 1 above)
Comment (ChristianKuehnel) [reply meeting_minutes/2021-08-31:55]: OK, I'll hook you up with my colleague via email.

**Pull request merged**